config/services.yaml

```diff
@@ -50,7 +50,11 @@ services:
     App\Service\RegistrationService:
         arguments:
             $appHost: '%app_host%'
-
+    
+    App\Service\SchedulingService:
+        arguments:
+            $entityManager: '@doctrine.orm.entity_manager'
```

Comment:
you don't need to inject entity manager service, it will be automatically injected, just put in constructor like this private readonloy EntityManagerInterface $entityManager, because symfony knows that it needs to autowire entityManager enttity for entityManagerInterface